manifests/c/ChilliBits/Spice/0.13.3/ChilliBits.Spice.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with WinGet Releaser v2 using Komac v1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ChilliBits.Spice
+PackageVersion: 0.13.3
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2023-03-15
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/spicelang/spice/releases/download/0.13.3/spice_x64_setup.msi
+  InstallerSha256: 1B00B6177981111E8D54A33331C298F023E8DB33CC2891E7F16A73D2C6D020AE

```

```suggestion
  InstallerSha256: 1B00B6177981111E8D54A33331C298F023E8DB33CC2891E7F16A73D2C6D020AE
  ProductCode: '{FD267129-7DE9-41A1-BEB0-19F1FCEEFE08}'
```
As it's an MSI installer, we should have ProductCode field in the manifest for better package matching.